example/track-to-rtp/main.go

```diff
@@ -61,110 +96,83 @@ func trackToRTP(track *webrtc.TrackRemote, receiver *webrtc.RTPReceiver) {
+		if !ok {
+			return
 		}
 
-		// Unmarshal the packet and update the PayloadType
-		if err = rtpPacket.Unmarshal(b[:n]); err != nil {
-			panic(err)
-		}
-		rtpPacket.PayloadType = c.payloadType
+		// Send a PLI on an interval so that the publisher is pushing a keyframe every rtcpPLIInterval
+		go func() {
+			ticker := time.NewTicker(time.Second * 2)
+			for range ticker.C {
 
-		// Marshal into original buffer with updated PayloadType
-		if n, err = rtpPacket.MarshalTo(b); err != nil {
-			panic(err)
-		}
+				// We need to add direct access to the peerconnection to ion-sdk-go to support PLI here
+				// PLI is disabled in this example currently
 
-		// Write
-		if _, err = c.conn.Write(b[:n]); err != nil {
-			// For this particular example, third party applications usually timeout after a short
-			// amount of time during which the user doesn't have enough time to provide the answer
-			// to the browser.
-			// That's why, for this particular example, the user first needs to provide the answer
-			// to the browser then open the third party application. Therefore we must not kill
-			// the forward on "connection refused" errors
-			if opError, ok := err.(*net.OpError); ok && opError.Err.Error() == "write: connection refused" {
-				continue
+				if rtcpErr := client.GetSubTransport().GetPeerConnection().WriteRTCP([]rtcp.Packet{&rtcp.PictureLossIndication{MediaSSRC: uint32(track.SSRC())}}); rtcpErr != nil {
+					fmt.Println(rtcpErr)
+				}
+			}
+		}()
+
+		b := make([]byte, 1500)
+		rtpPacket := &rtp.Packet{}
+		for {
+			// Read
+			n, _, readErr := track.Read(b)
+			if readErr != nil {
+				panic(readErr)
 			}
-			panic(err)
-		}
-	}
-}
-
-func main() {
-	// init log
-	log.Init("debug")
 
-	// parse flag
-	var session, addr, file string
-	flag.StringVar(&file, "file", "./file.webm", "Path to the file media")
-	flag.StringVar(&addr, "addr", "localhost:50051", "Ion-sfu grpc addr")
-	flag.StringVar(&session, "session", "test session", "join session name")
-	flag.Parse()
+			// Unmarshal the packet and update the PayloadType
+			if err = rtpPacket.Unmarshal(b[:n]); err != nil {
+				panic(err)
+			}
+			rtpPacket.PayloadType = c.payloadType
 
-	// add stun servers
-	webrtcCfg := webrtc.Configuration{
-		ICEServers: []webrtc.ICEServer{
-			webrtc.ICEServer{
-				URLs: []string{"stun:stun.stunprotocol.org:3478", "stun:stun.l.google.com:19302"},
-			},
-		},
-	}
+			// fmt.Println("rtpPacket", rtpPacket)
 
-	config := sdk.Config{
-		Log: log.Config{
-			Level: "debug",
-		},
-		WebRTC: sdk.WebRTCTransportConfig{
-			Configuration: webrtcCfg,
-		},
-	}
-	// new sdk engine
-	e := sdk.NewEngine(config)
+			// Marshal into original buffer with updated PayloadType
+			if n, err = rtpPacket.MarshalTo(b); err != nil {
+				panic(err)
+			}
 
-	// create a new client from engine
-	c, err := sdk.NewClient(e, addr, "client id")
-	if err != nil {
-		log.Errorf("err=%v", err)
-		return
+			// Write
+			if _, err = c.conn.Write(b[:n]); err != nil {
```

Comment:
When I originally wrote this sample, I also copied this same block of code from `pion/webrtc/examples/rtp-forwarder` but that example is based off a JSFiddle approach and a CLI interaction - I don't believe this block of code belongs in this example anymore.